src/main/java/ninjabrainbot/io/KeyPresser.java

```diff
@@ -28,15 +28,13 @@ public void paste(String s) throws InterruptedException {
 		StringSelection selection = new StringSelection(s);
 		clipboard.setContents(selection, null);
 		robot.keyPress(CTRL_KEYCODE);
-		Thread.sleep(50);
 		robot.keyPress(KeyEvent.VK_V);
+		Thread.sleep(60);
 		robot.keyRelease(CTRL_KEYCODE);
-		Thread.sleep(50);
```

Comment:
I do not want to remove these delays, they were added to make calibration work for linux https://github.com/Ninjabrain1/Ninjabrain-Bot/pull/80/files#diff-4a7decb39036c3449c451008448d3a5ff4da95826e66383bfb329150388e5d78

In general I do not want to remove delays, calibration has been so buggy in the past and its fine to make the calibration a fraction of a second longer if it means it is more stable

Reply:
Sure, I'll add them back. For reference, the 60ms delay between `robot.keyPress(KeyEvent.VK_V)` and `robot.keyRelease(CTRL_KEYCODE)` is needed prior to 1.13.